File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Models/SeatAvailableQuantity.cs
using System;

namespace ConferenceManagement
{
    public class SeatAvailableQuantity
    {
        public SeatAvailableQuantity(Guid seatTypeId, int availableQuantity)
        {
            SeatTypeId = seatTypeId;
            AvailableQuantity = availableQuantity;
        }

        public Guid SeatTypeId { get; private set; }
        public int AvailableQuantity { get; private set; }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Models/SeatAvailableQuantity.java
package com.microsoft.conference.management.domain.Models;

public class SeatAvailableQuantity {
    public String SeatTypeId;
    public int AvailableQuantity;

    public SeatAvailableQuantity(String seatTypeId, int availableQuantity) {
        SeatTypeId = seatTypeId;
        AvailableQuantity = availableQuantity;
    }
}
